src/Utility/PlayerMovementGrid.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * (c) Vladimir "allejo" Jimenez <me@allejo.io>
+ *
+ * For the full copyright and license information, please view the
+ * LICENSE.md file that was distributed with this source code.
+ */
+
+namespace App\Utility;
+
+class PlayerMovementGrid
+{
+    /** @var array */
+    private $movement;
+    /** @var int Size of the map in the replay file */
+    private $grid_size;
+    /** @var int Size of the heatmap */
+    private $heatmap_size;
```

Comment:
```suggestion
    /** @var int[][] */
    private $movement;
    /** @var int Size of the map in the replay file */
    private $worldSize;
    /** @var int Size of the heatmap */
    private $cellSize;
```

camelCase and let's rename `grid_size` to `worldSize` and `heatmap_size` to `cellSize`

Reply:
renaming `heatmap_size` to `heatmapSize`